test/jasmine/tests/shapes_test.js

```diff
@@ -103,75 +103,75 @@ describe('Test shapes defaults:', function() {
     });
 });
 
-describe('Test shapes:', function() {
-    'use strict';
+function countShapesInLowerLayer(gd) {
+    return gd._fullLayout.shapes.filter(isShapeInLowerLayer).length;
+}
```

Comment:
Moved these helpers to the root level so I could use them in a different `describe` block (which wants to set up a different plot in `gd`)